README.md

```diff
@@ -12,7 +12,7 @@ Visit the [Commerce Cloud Developer Center](https://developer.commercecloud.com/
 Download and install Node.js and npm [here](https://nodejs.org/en/download/).
 ​
 
-> **Note:** Only Node.js version 10 and 12 LTS are supported. Other versions can cause unexpected results. To use a different version of Node.js for other projects, you can manage multiple versions of Node.js with [nvm](https://github.com/nvm-sh/nvm).
+> **Note:** Only Node.js version 10, 12, 14 and 16 LTS are supported. Other versions can cause unexpected results. To use a different version of Node.js for other projects, you can manage multiple versions of Node.js with [nvm](https://github.com/nvm-sh/nvm).
```

Comment:
* We should drop official support for v10, since it's EOL. We can keep it in CI to support it unofficially.
* We don't necessarily need to start officially supporting v16 immediately. Like we did for v14, we can add it to CI and support it unofficially until it's Active LTS.